chef/cookbooks/crowbar-pacemaker/recipes/drbd.rb

```diff
@@ -26,7 +26,10 @@
 
 if claimed_disks.empty? and not unclaimed_disks.empty?
   unclaimed_disks.each do |disk|
-    if disk.claim(claim_string)
+    %x{lsblk #{disk.name} --noheadings --output MOUNTPOINT | grep -q -v ^$}
```

Comment:
I agree with @toabctl that we should fix it in BarclampLibrary; which probably means we would need to handle the bootdisk magic differently, but that's certainly doable.